helm/provisioner/Chart.yaml

```diff
@@ -1,16 +1,14 @@
-apiVersion: v1
-name: provisioner
-description: local provisioner chart
+apiVersion: v2
```

Comment:
it looks like you're making it compatible with helm3, is the presubmit Github action job, are there any other changes in the this bump? would it be tested through the kind cluster that you're trying to setup in a presubmit job?

Reply:
More context: 

```
      - name: Create Kind cluster
        uses: helm/kind-action@9e8295d178de23cbfbd8fa16cf844eec1d773a07
```

Reply:
yeah makes sense to catch bugs in a kind cluster too. Thanks